src/fontMetrics.js

```diff
@@ -252,12 +264,14 @@ const getCharacterMetrics = function(character, style) {
     }
 };
 
-const fontMetricsBySizeIndex = {};
+export type FontMetrics = {[charCode: number]: Array<number>};
```

Comment:
I recommend something along the lines of
```javascript
type FontSizeIndex = 0 | 1 | 2;
type FontSizeMetrics = {[string]: number};
const fontMetricsBySizeIndex: {[FontSizeIndex]: FontSizeMetrics} = {};
```
and using `let sizeIndex: FontSizeIndex` below. The now defunct [PR 853](https://github.com/Khan/KaTeX/pull/853) demonstrates this.